### ISSUE TITLE
Render image in codeblock

### ISSUE THREAD

**Issue by dennisseidel** [JeppeKlitgaard/ObsidianAnkiBridge#20]
Great plugin and well done! :) One convenience feature I realized during my usage is: 

I'd like to see that images included in codeblock cards render in the editor. E.g. 

```anki
id: 1644017054779
---

## What Does a Program Do?

![[Pasted image 20220204021645.png]]
```
Renders as 

![image](https://user-images.githubusercontent.com/18036223/152640000-78ec8f15-d305-49cf-8254-6e1abae7c4fc.png)

 I would be great if the plugin could display the image in the codeblock preview of the editor

Currently the workaround is to post the image again under the codeblock.

**Comment by dennisseidel** [comment]
I see that @nicojeske works on this functionality in his fork :)  https://github.com/nicojeske/ObsidianAnkiBridge/commits/master

**Comment by JeppeKlitgaard** [comment]
Hey @dennisseidel 

Thank you for your kind words!

This is an issue I am aware of (and something that bothers me as well), and I was hoping a fix would be made directly in Obsidian quite soon – other embeds (Audio, Video) also suffer from the same problem.

For now you can try @nicojeske's PR and see if that works – I have an exam coming up soon, so I might not get around to merging before a couple of weeks from now.

**Comment by JeppeKlitgaard** [comment]
This has been fixed by @nicojeske and released in `0.8.0`. Apologies for the wait!
